test.properties

```diff
@@ -439,7 +439,7 @@
     database.mysql.service.cmd.stop=mysqld stop
     database.mysql.service.executable=service
     database.mysql.username=root
-    database.mysql.url=jdbc:mysql://${database.mysql.host}/${database.mysql.schema}?characterEncoding=UTF-8&dontTrackOpenResources=true&holdResultsOpenOverStatementClose=true&serverTimezone=GMT&useFastDateParsing=false&useUnicode=true
+    database.mysql.url=jdbc:mysql://${database.mysql.host}/${database.mysql.schema}?characterEncoding=UTF-8&dontTrackOpenResources=true&holdResultsOpenOverStatementClose=true&serverTimezone=GMT&useFastDateParsing=false&useUnicode=true?permitMysqlScheme
```

Comment:
@Kvale1733 Can you confirm whether it should be `?permitMysqlScheme` or `&permitMysqlScheme` or both work? Usually `?` is the indicator of the beginning of the query string path in a URL, while the parameters are separated by `&`, and this URL already has a lot of parameters.

cc @vicnate5

Reply:
@Kvale1733 Also, could you make it more specific in the commit message about the reason that we need to add this parameter? I mean, just looking at these properties, it seems to be used together with MySQL's own driver. So we'd better point out if there are tests that use this URL, but with the MariaDB driver name in `database.mysql.driver` or something like that.

Reply:
On it.